orm/orm.go

```diff
@@ -255,14 +252,15 @@ func GetUnsentMessages(db *sql.DB, maxCnt int) []QueuedMessage {
 }
 
 func getChatRowIDByToken(tx *sql.Tx, token string) (int, error) {
-	row := tx.QueryRow("SELECT rowid FROM Chats WHERE token = $1", &token)
+	row := tx.QueryRow("SELECT internal_id FROM Chats WHERE token = $1", &token)
 	rowID := -1
 	err := row.Scan(&rowID)
 	return rowID, err
 }
 
 // Subscribes proveded chat on another with given token.
-//  Returns true on success
+//
```

Comment:
This empty line seems to have no effect

Reply:
it is added automatically by extension, but no idea why